example/api_ref.py

```diff
@@ -0,0 +1,67 @@
+import os
+import json
+import boto3
+import pandas as pd
+from sm_annotation_utils import sm_annotation_utils
+def get_study_info(ds_ids, database='HMDB', fdr=0.1, config=None):
```

Comment:
Should database still have 'HMDB' value? @andy-d-palmer

Reply:
well spotted - fixed